src/drivers/barometer/ms5611/ms5611_i2c.cpp

```diff
@@ -253,3 +256,15 @@ MS5611_I2C::_read_prom()
 	/* calculate CRC and return success/failure accordingly */
 	return (ms5611::crc4(&_prom.c[0]) && !bits_stuck) ? PX4_OK : -EIO;
 }
+
+int MS5611_I2C::init()
+{
+	int ret = I2C::init();
+
+	if (ret != PX4_OK) {
+		PX4_ERR("I2C::init failed");
+		return ret;
+	}
+
+	return probe();
```

Comment:
Actually, this is can and should be fixed lower level. In the NuttX version of the I2C wrapper `I2C::init()` calls probe. https://github.com/PX4/Firmware/blob/cc7a0cd69dec52db7816a24e58e782a82a8243a3/src/lib/drivers/device/nuttx/I2C.cpp#L140

I'll bring the posix version into sync. https://github.com/PX4/Firmware/blob/cc7a0cd69dec52db7816a24e58e782a82a8243a3/src/lib/drivers/device/posix/I2C.cpp#L74

Reply:
https://github.com/PX4/Firmware/pull/13818

Reply:
Can you delete this now and retest? https://github.com/PX4/Firmware/pull/13818 has merged.